### PR TITLE
Update background colour for search buttons

### PR DESCRIPTION
**Pull request by alex-ju**
## What
This is the only place where we use the `light-blue` in the library and as far as I know, for no strong reasons.

## Why
Using `blue` increases the contrast with the magnifying glass icon and makes the component more consistent with other elements on the page.

## Visual Changes
Before
Screen Shot 2019-11-14 at 14 14 34

After
Screen Shot 2019-11-14 at 14 14 15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Update background colour for search buttons ([PR #1197](https://github.com/alphagov/govuk_publishing_components/pull/1197))
```
Comment (DilwoarH) [inline CHANGELOG.md]: Do we need this for a guidance page update?

**Comment by DilwoarH** [comment]
This component isn't being used anywhere:

https://github.com/search?q=org%3Aalphagov+%22govuk_publishing_components%2Fcomponents%2Fsearch%22&type=Code

Should we investigate?

**Comment by andysellick** [comment]
Screenshot 2019-11-14 at 15 51 02

Must be something weird with github's search, it's definitely used ☝️

**Comment by DilwoarH** [comment]
> Screenshot 2019-11-14 at 15 51 02
> 
> Must be something weird with github's search, it's definitely used ☝️

Yes ignore me, my search query was bad. It's used here:
https://github.com/alphagov/frontend/blob/master/app/views/homepage/index.html.erb#L18

**Comment by alex-ju** [comment]
> This component isn't being used anywhere:

The default version is used on [the search page ](https://www.gov.uk/search)(don't ask how to get there though) 😬

**Pull request merged**